action.yml

```diff
@@ -18,6 +18,12 @@ outputs:
   terraform-docs-version:
     description: The version of terraform-docs to use.
     value: ${{ steps.terraform-docs.outputs.version }}
+  go-mock-version:
+    description: The version of go mock to use.
+    value: ${{ steps.go-mock-version.outputs.version }}
+  go-junit-report-version:
+    description: The version of go junit report to use.
+    value: ${{ steps.go-junit-report.outputs.version }}
```

Comment:
Please reorder these to match the order in the steps below this in the file.

Reply:
Sorry your suggestion came after my commit. Please see commit ecbc0939d1f16bce6d60cf088943fd6904f530dc